sips/sip-x/sip-x-nft-metadata.md

```diff
@@ -187,16 +187,42 @@ pt-BR.json
 ```
 
 ### Properties
-Common properties are 
+Common properties are described in Appendic C.
 
-| Name | Type | Description |
-|------|------|-------------|
-| `collection` | `string` | collection name the token belongs to. See also Appendix A. |
-| `decimals` | `integer` | number of decimals. See also Appendix A. |
-| `id` | `integer` | identifier for NFTs. See also Appendix A. |
-| `created` | `integer` | creation date of the token in unix timestamp | 
-| `symbol`  | `string` | token symbol |
-| `ip_document_uri` | `string` | link to document about intellectual property (IP) rights |
+### Properties
+Common properties are described in appendix C. Properties of type object are usually described using the following schema:
+```
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Token Metadata Property",
+    "type": "object",
+    "required": ["value"],
+    "properties": {
+        "type": {
+            "type": "string",
+            "description": "type of the property"
+        },
+        "description": {
+            "type": "string",
+            "description": "description of the property"
+        },
+        "value": {
+            "type": any,
+            "description": "value of the property"
+        }
+    }
+}
+```
+
+Example:
+
+```
+{
+    "type": "string", 
+    "description": "Address of custodian key holder", 
+    "value": "Casa Inc., P.O. Box 20575, Charleston, S.C. 29413, United States."
```

Comment:
good this is extensible this far, and not too complex... but i would rather focus on digital assets first.

Reply:
In general disagree with focusing on digital assets only.

Reply:
This is only an example, even for digital assets there might be meta data that we don't want to specify here like `camera` or `iso`, ...